__init__.py

```diff
@@ -621,9 +622,9 @@ def handle_next_weekend_weather(self, message):
         """ Handle next weekends weather """
 
         report = self.__initialize_report(message)
-        when, _ = self.__extract_datetime('next saturday', lang='en-us')
+        when, _ = self.__extract_datetime('next saturday', lang=self.lang)
```

Comment:
will this work? next saturday is english?

Reply:
no it will not. nice eye. that's unfortunate, and will require some thought.

Reply:
I think we should be loading English by default in Mycroft-core for the foreseeable future, hence it should always be available.
This can then remain as `en-us` for the moment as it's only intended to get a datetime object.

Reply:
That's fair.